src/utils/jwk.ts

```diff
@@ -37,3 +37,26 @@ export const JWK = z.object({
   /** JWK "x5u" (X.509 URL) Parameter. */
   x5u: z.string().optional(),
 });
+
+/**
+ * Ensure key values are encoded using base64url and not just base64, as defined in https://datatracker.ietf.org/doc/html/rfc7517
+ *
+ * @see https://datatracker.ietf.org/doc/html/rfc7517
+ *
+ * @param key The key to fix
+ * @returns THe same input key with fixed values
+ */
+export function fixBase64EncodingOnKey(key: JWK): JWK {
```

Comment:
what do you think about adding this function to `io-react-native-jwt` ?

Reply:
I was thinking more on `io-react-native-crypto`. This is actually a deviation from the specification of any key in jwk format, it may concern all keys in the app. 

I'd leave it here for now